postgres/initdb.sql

```diff
@@ -43,7 +43,7 @@ CREATE TABLE IF NOT EXISTS tasks (
     project_id varchar,
     group_id varchar,
     task_id varchar,
-    geom geometry(MULTIPOLYGON, 4326),
+    geom geometry,
```

Comment:
I think that tasks should still be a multi polygon. or do we have any case where this is not true?

Reply:
true! I will change that!